### PR TITLE
Update gradle-dev.yml

### PR DESCRIPTION
**Pull request by wysohn**

### CODE REVIEW

File: .github/workflows/gradle-dev.yml [hunk 1]
```diff
@@ -121,10 +121,12 @@ jobs:
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+        
       - name: Build with Gradle
         run: ./gradlew build -x test
+        
       - name: Cache Build
-      - uses: actions/cache@v2
+        uses: actions/cache@v2
```
Comment (Ioloolo) [inline .github/workflows/gradle-dev.yml:129]: oops.. my mistake at #468

**Review by son-daehyeon — APPROVED** [review]

File: .github/workflows/gradle-dev.yml (hunk 1 above)
Comment (wysohn) [reply .github/workflows/gradle-dev.yml:129]: @Ioloolo 
No, I'm just asking your opinion

I didn't get the chance to test it

Seems to be working again though

Comment (Ioloolo) [reply .github/workflows/gradle-dev.yml:129]: IMO, Your change looks better.

Naming procedures will be easy to recognize where an error occur if it occurred.

Comment (Ioloolo) [reply .github/workflows/gradle-dev.yml:129]: You look very busy!

Comment (wysohn) [reply .github/workflows/gradle-dev.yml:129]: > You look very busy!

Indeed xD

**Pull request merged**